node/packages/aws-lambda-otel-extension/internal/otel-extension-internal-node/index.js

```diff
@@ -198,26 +198,23 @@ const responseHandler = async (span, { res, err }, isTimeout) => {
   if (isTimeout) {
     functionData.error = true;
     functionData.errorCulprit = 'timeout';
-    functionData.errorExceptionType = 'TimeoutError';
-    functionData.errorExceptionMessage = '';
-    functionData.errorExceptionStacktrace =
-      'Function execution exceeded the configured timeout limit.';
+    functionData.errorType = 'TimeoutError';
   } else if (err) {
     functionData.error = true;
     functionData.errorCulprit = err.message;
-    functionData.errorExceptionType = typeof err;
-    functionData.errorExceptionMessage = err.message;
-    functionData.errorExceptionStacktrace = err.stack;
+    functionData.errorType = typeof err;
+    functionData.errorMessage = err.message;
+    functionData.errorStacktrace = err.stack;
   } else if (
     pathData['http.status_code'] >= 500 &&
     ['aws.apigateway.http', 'aws.apigatewayv2.http'].includes(functionData.eventType)
   ) {
     // This happens if we get a 500 status code set explicity within in the app
     functionData.error = true;
     functionData.errorCulprit = 'internal server error';
-    functionData.errorExceptionType = typeof new Error();
-    functionData.errorExceptionMessage = 'internal server error';
-    functionData.errorExceptionStacktrace = 'internal server error';
+    functionData.errorType = typeof new Error();
```

Comment:
And then here maybe out “handled”?

Reply:
I think both of them are handled.

What's the real difference? Is that one is response for API Gateway? and other is not?

Reply:
Ok, I looked closer. 

Second is when we report `500` with success resolution.

I guess it's a situation where the user app might have crashed and HTTP decorator converted it to `500`which is reported with _success_ response (so not really an error).

That's an interesting case, as technically, just by looking at lambda response, it's a _success_  resolution.

Reply:
In this if block the user is returning a `500` error response from their API.

The if statement above is one where you would throw an error from an async function or pass something like `callback(new Error(), null)` from your lambda function

Reply:
@medikoo yeah exactly 👍 For context, this was a requirement that successful lambda invocations that return a 500 error as an error so they would show up as an error in console.